lite/verifying/provider_test.go

```diff
@@ -13,10 +13,10 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
-func TestInquirerValidPath(t *testing.T) {
+func TestProviderValidPath(t *testing.T) {
 	assert, require := assert.New(t), require.New(t)
-	trust := NewDBProvider("trust", dbm.NewMemDB())
-	source := NewDBProvider("source", dbm.NewMemDB())
+	trust := lite.NewDBProvider("trust", dbm.NewMemDB())
+	source := lite.NewDBProvider("source", dbm.NewMemDB())
```

Comment:
Import is missing (test file doesn't compile):
add "github.com/tendermint/tendermint/lite"

Reply:
Also, in the following lines `genPrivKeys` is used which doesn't exist in this package. It does in "github.com/tendermint/tendermint/lite"